Library/Formula/cryptol.rb

```diff
@@ -0,0 +1,53 @@
+require "language/haskell"
+
+class Cryptol < Formula
+  include Language::Haskell::Cabal
+
+  homepage "http://www.cryptol.net/"
+  url "https://github.com/GaloisInc/cryptol/archive/v2.1.0.tar.gz"
+  sha1 "b0432097c807518eafe138c77a02d8142cecc223"
+  head "https://github.com/GaloisInc/cryptol.git"
+
+  depends_on "ghc" => :build
+  depends_on "cabal-install" => :build
+  depends_on "cvc4"
+
+  # patch to allow parallelized builds
+  patch :DATA
```

Comment:
Yep, in fact it's no longer a problem on HEAD of cryptol, so I should probably change the formula to only apply it to stable.